parts/kubernetesmastercustomdata.yml

```diff
@@ -7,8 +7,30 @@ packages:
  - jq
  - traceroute
 
+disk_setup:
+  /dev/sdc:
+      table_type: gpt
+      layout: true
+      overwrite: true
+
+fs_setup:
+- special:
+  cmd: /sbin/mkfs.ext4 /dev/sdc1 -F -E lazy_itable_init=1,lazy_journal_init=1
```

Comment:
Is this the ultimate fix for the overall issue? How fragile is this? Is there a longer term fix for there being some sort of race condition here?